[4.6] Doc for Elasticsearch 8 (#3022)
ezsystems/developer-documentation: docs/update_and_migration/from_4.6/update_to_5.0.md

;
```

#### Remove separate Elasticsearch 8 package

If you were using the separate `ibexa/elasticsearch8` package in v4.6, you should switch back to the built-in `ibexa/elasticsearch` package, as it now supports both Elasticsearch 7 and Elasticsearch 8.

```bash
composer remove --no-update ibexa/elasticsearch8
```

> The `ibexa/elasticsearch` package will be automatically installed as part of your [[= product_name =]] 5.0 update.

Reviewer: ```suggestion
The `ibexa/elasticsearch` package is automatically installed as part of your [[= product_name =]] 5.0 update.
```

> Your existing Elasticsearch 8 server and configuration will continue to work with the `ibexa/elasticsearch` package.

Reviewer: ```suggestion
Your existing Elasticsearch 8 server and configuration continue to work with the `ibexa/elasticsearch` package.
```
